src/NServiceBus.Core/Serializers/SerializationFeatureHelper.cs

```diff
@@ -1,19 +1,41 @@
 ﻿namespace NServiceBus.Features
 {
+    using System.Collections.Generic;
+    using System.Linq;
+    using NServiceBus.Serialization;
+
     /// <summary>
     /// Base class for all serialization <see cref="Feature"/>s.
     /// </summary>
     public static class SerializationFeatureHelper
     {
         /// <summary>
-        /// Allows serialization features to verify their <see cref="Feature"/> Prerequisites.
+        /// Allows serialization features to verify their <see cref="ConfigureSerialization"/> Prerequisites.
         /// </summary>
-        public static bool ShouldSerializationFeatureBeEnabled(this Feature serializationFeature, FeatureConfigurationContext context)
+        public static bool IsDefaultSerializer(this ConfigureSerialization serializationFeature, FeatureConfigurationContext context)
```

Comment:
this is a breaking change. do we need an obsolete or things added to the upgrade guide?